ui/page/root/home_page.go

```diff
@@ -756,11 +756,6 @@ func (hp *HomePage) startSyncing(wallet sharedW.Asset, unlock load.NeedUnlockRes
 						log.Debugf("Error starting sync: %v", err)
 					}
 
-					if hp.AssetsManager.IsHTTPAPIPrivacyModeOff(libutils.ExchangeHTTPAPI) {
```

Comment:
our api keys have rate limits https://github.com/crypto-power/cryptopower/issues/193, so we should not be auto syncing because of this reason, users should manually trigger sync updates if they need to, or click on an order to update it